apps/dsky_clock/metadata.json

```diff
@@ -0,0 +1,17 @@
+{ "id": "dsky_clock",
+    "name": "DSKY Clock",
+    "shortName":"dsky_clock",
+    "icon": "dsky_clock.img",
+    "screenshots": [{"url":"screenshot1.png"}],
+    "version": "0.01",
+    "description": "A clockface inspired by the Apollo DSKY interface.",
+    "tags": "clock",
+    "type": "clock",
+    "supports":["BANGLEJS2"],
+    "readme": "README.md",
+    "allow_emulator": true,
+    "storage": [
+      {"name":"app.js","url":"app.js"},
+      {"name":"dsky_clock.img","url":"app-icon.js","evaluate":true}
```

Comment:
You'll have to add an `app-icon.js` file to the files in the repo dsky_clock folder - currently this links to non-existing file.